go.mod

```diff
@@ -9,11 +9,11 @@ require (
 	github.com/aristanetworks/goarista v0.0.0-20200609010056-95bcf8053598 // indirect
 	github.com/centrifuge/go-substrate-rpc-client/v2 v2.0.1
 	github.com/ethereum/go-ethereum v1.9.25
-	github.com/google/addlicense v0.0.0-20210428195630-6d92264d7170 // indirect
```

Comment:
Why did you remove this?

Reply:
It was removed automatically by the command rub `go mod tidy`